lost/src/main/AndroidManifest.xml

```diff
@@ -15,7 +15,14 @@
         android:required="false" />
     <uses-feature android:name="android.hardware.wifi" />
 
-    <!-- Wow. This is insane. You require an empty tag. -->
-    <application />
+
+    <uses-permission android:name="android.permission.BLUETOOTH"/>
```

Comment:
I'm concerned about including bluetooth permissions in the library manifest. Developers using lost are typically sensitive (and rightfully so) about adding unnecessary permissions to their apps.

Would it be possible to exclude these permissions from the library manifest? Then we could specify in the README if you want to access bluetooth settings using lost you must add these permissions to the app manifest.